src/rust/engine/src/scheduler.rs

```diff
@@ -451,132 +471,34 @@ impl Scheduler {
     let (sender, receiver) = mpsc::channel();
     self.execute_helper(request, session, sender);
 
-    // This map keeps the k most relevant jobs in assigned possitions.
-    // Keys are positions in the display (display workers) and the values are the actual jobs to print.
-    let mut tasks = IndexMap::new();
+    let interval = Duration::from_millis(1000 / ConsoleUI::render_rate_hz());
```

Comment:
This would benefit from a comment. It looks like we previously always used 100ms no matter what.

Reply:
That hard-coding of 100 ms is now done in `render_rate_hz` rather than `scheduler.rs`, but it's still just a hard-coded reasonable value.

Reply:
Right, but this code will be wrong if `render_rate_hz` changes, right? It would probably be good to mention that coupling in a comment, e.g. in `render_rate_hz` to add 

```rust
// Update scheduler.rs when changing this value.
```

Reply:
"hertz" is "per second"... so dividing 1000 by hertz always gets you millis. The goal here was to prevent things going out of sync.

Reply:
Oh. True. Good point. Okay no need for a comment!